zkevm-circuits/src/evm_circuit/param.rs

```diff
@@ -7,7 +7,7 @@ use halo2_proofs::{
 use std::collections::HashMap;
 
 // Step dimension
-pub(crate) const STEP_WIDTH: usize = 140;
+pub(crate) const STEP_WIDTH: usize = 178;
```

Comment:
is it possible to increase MAX_STEP_HEIGHT instead of increasing STEP_WIDTH?    Besides, i guess after increasing MAX_STEP_HEIGHT, we can make N_BYTES_LOOKUPS smaller, maybe from 64 to ~40?

Reply:
reverted back this change